src/modules/cspec/CMakeLists.txt

```diff
@@ -1,58 +1,12 @@
-# Copyright (C) 2021 European Spallation Source, ERIC. See LICENSE file
-
-add_subdirectory(test)
-add_subdirectory(generators)
-include_directories(.)
+# Copyright (C) 2022 European Spallation Source, ERIC. See LICENSE file
```

Comment:
2022 - 2023 ?
If this started in 2022, though, else just 2023

Reply:
Another casualty from excessive `sed` usage.